### ISSUE TITLE
[Warning] The web page should have the content's human language indicated in the markup (2)

### ISSUE THREAD

**Issue by rodowi** [mxabierto/participa#234]
/participa

https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_html_01

```html
<html><head>...</html>
```html
